src/MarkovChainMonteCarlo.jl

```diff
@@ -214,14 +221,20 @@ end
 
 
 function proposal(mcmc::MCMC)
-
     proposal_covariance = get_cov(mcmc.prior)
- 
+    prop_dist = MvNormal(zeros(length(mcmc.param)),
+                         (mcmc.step[1]^2) * proposal_covariance)
+
     if mcmc.algtype == "rwm"
-        prop_dist = MvNormal(zeros(length(mcmc.param)), 
-                             (mcmc.step[1]^2) * proposal_covariance)
+        sample = mcmc.posterior[:,1 + mcmc.iter[1]] .+ rand(prop_dist)
+    elseif mcmc.algtype == "pCN"
+        # 0 < ρ < 1 technically a free parameter; tie to stepsize following
+        # Beskos et al (2017) "Geometric MCMC for infinite-dimensional inverse problems."
+        ρ = (1 - mcmc.step[1]/4) / (1 + mcmc.step[1]/4)
+        sample = ρ * mcmc.posterior[:,1 + mcmc.iter[1]] .+ sqrt(1 - ρ^2) * rand(prop_dist)
```

Comment:
This is fine. Though you reference Beskos here, I think all that is required here is a map from `f:[0,infty) -> [0,1)` where `f(mcmc.step) = \rho`

Reply:
Wording changed in d8183f8 : agree that your condition is necessary, but (unless I was misunderstanding) it's additionally constrained by the (time) discretization scheme used for the Langevin SDE; i.e. if something more complicated than Euler was used, the relation of that stepsize to the SDE's ρ would also need to be more complex. 

At any rate, the intention of the comment was to give the reader a link to follow if they want to know where the equation came from; changed the wording to make this clearer.

Reply:
I think the beskos one is indeed constrained - but in the MALA case you have the interpretation of the Langevin SDE - here I do not think there is such a clear interpretation?